types/session_cost.yaml

```diff
@@ -1,5 +1,62 @@
 description: Types to provide the session price
 types:
+  IdlePrice:
+    description: >-
+      Price components when idle
+    type: object
+    additionalProperties: false
+    properties:
+      grace_minutes:
+        description: >-
+          Grace period in minutes before idle time is charged.
+          Grace minutes start counting from the timestamp of the message in which state
+          changed from "Charging" to "Idle".
+        type: integer
+      hour_price:
+        description: >-
+          Price per hour while idle
+        type: number
+  CostCategory:
+    description: Cost category
+    type: string
+    enum:
+      - Energy
```

Comment:
can't we model the idle price not as a category in here?

Reply:
So you would like to add a IdlePriceTime, ChargingTime, IdleGraceMinutes and remove Time? I thought about it but I don't know if that makes things more clear.

Reply:
i don't really understand why we need to define actually idle as a separate category. Your linked whitepater says
> It is up to the CSMS and not the charging station, to decide when a transaction is considered to be in idle time.

So at some point when the station sends a `chargingState` or `atPowerkW` and the CSMS thinks that its now idle it will send a price containing just the time component. The CSMS can internally also account for grace times or other conditions

Reply:
Yes but what if the CSMS is not online? 
I just reflect the fields from the California Pricing Requirements, which sends the idle price as well.